pkg/drivers/hyperv/hyperv.go

```diff
@@ -0,0 +1,509 @@
+package hyperv
```

Comment:
woud need to add boilerplate copyright here and other new files

Reply:
Do we need to keep the docker copyright? This code will quickly change and docker has abandoned this code years ago.

We can add a note that this was imported from the docker project.